onnxruntime/core/framework/sequential_executor.cc

```diff
@@ -22,6 +22,11 @@ static Status ReleaseNodeMLValues(ExecutionFrame& frame,
                                   const SequentialExecutionPlan::NodeExecutionPlan& node_exec_plan,
                                   const logging::Logger& logger);
 
+static inline std::string NodeIdForLoggingPurpose(const OpKernel* p_op_kernel) {
+  return "\"" + (p_op_kernel->Node().Name().empty() ? p_op_kernel->Node().OpType() : p_op_kernel->Node().Name()) +
```

Comment:
Prefer += (append) or ostringstream to concatenate multiple strings. In this case, this function is getting called only when exceptions are thrown hence it's not so much of a concern. However, given that the function is available outside Execute, it could potentially be used in other contexts as well.

Reply:
It's only "ForLoggingPurpose". It shouldn't be used in any scenario that has perf concern.

Reply:
Can we always include the OpType as I think that provides important information?

FWIW we have MakeString in common.h which is used by the macros as a slightly nicer way to use ostringstream. Just pass a common separated list of the values to combine.